docs/Tutorials/Python.md

```diff
@@ -24,3 +24,41 @@ supports `pip`, for instance in a
 access to the SpECTRE Python modules by adding  `BUILD_DIR/bin/python` to your
 `PYTHONPATH`. This is done automatically by sourcing the `LoadPython.sh` file
 with the command `. BUILD_DIR/bin/LoadPython.sh`.
+
+## Running Jupyter within the Docker container
+
+[Jupyter lab](https://jupyterlab.readthedocs.io/) is installed in the Docker
+container. You can run it in the container and access it through a browser on
+the host for a convenient way to work with the SpECTRE Python bindings. To do
+so, make sure you have exposed a port when running the Docker container, e.g.
+by appending the option `-p 8000:8000` to the `docker run` command (see
+\ref installation). Inside the docker container, it can be convenient to
+`apt-get install screen` and use `screen` to obtain a shell that runs the
```

Comment:
Why prefer screen over disowning the job?

Reply:
No reason, only a suggestion. Do you prefer to remove this sentence?

Reply:
No, seems useful. Maybe add disowning as an alternative screen? screen can be a bit awkward to use when you first get started.

Reply:
ok I'm namedropping `disown` now so people can look it up when they are interested (same for `screen`)